ferrischat_webserver/src/channels/delete_channel.rs

```diff
@@ -43,7 +41,7 @@ pub async fn delete_channel(req: HttpRequest, _: crate::Authorization) -> impl R
         channel: channel_obj.clone(),
     };
 
-    if let Err(e) = fire_event(format!("channel_{}_{}", channel_id, guild_id), &event).await {
+    if let Err(e) = fire_event(format!("channel_{}", channel_id), &event).await {
```

Comment:
```suggestion
    if let Err(e) = fire_event(format!("channel_{}_{}", channel_id, channel_obj.guild_id), &event).await {
```
We only listen to events with the guild ID, so removing it would essentially send this event to the void.